scripts/setup-macos.sh

```diff
@@ -43,6 +43,7 @@ MACOS_VELOX_DEPS="bison flex gflags glog googletest icu4c libevent libsodium lz4
 MACOS_BUILD_DEPS="ninja cmake"
 FB_OS_VERSION="v2024.09.16.00"
 FMT_VERSION="10.1.1"
+FAST_FLOAT_VERSION="v6.1.6"
```

Comment:
Local quick workaround is to install it through brew or use folly bundling. How did I miss this.

Reply:
What I don't get was why the CI for MacOS succeeded before. Could it be the FAST_FLOAT_VERSION variable was set in Linux and somehow the value remains?